config/application.rb

```diff
@@ -14,5 +14,14 @@ class Application < Rails::Application
     # Settings in config/environments/* take precedence over those specified here.
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
+
+    Dotenv.load  # We need this to perform the checks below.
+    review_app_pattern = /^library-thesis-dropbox-(staging-)?pr-[\d]+$/
```

Comment:
All PR builds should be based on the staging app so the parens should be safe to remove.